UpgradedVehicles/QPatch.cs

```diff
@@ -61,6 +61,7 @@ public static void Patch()
 
         public static void CrossModUpdates()
         {
+#if SUBNAUTICA
```

Comment:
I actually have thoughts myself about getting the seamoth working in BZ so maybe not disabling the seamoth checks.